client/components/Events/ManageEventTemplatesModal.jsx

```diff
@@ -0,0 +1,205 @@
+/* eslint-disable react/no-multi-comp */
+import React from 'react';
+import PropTypes from 'prop-types';
+import {connect} from 'react-redux';
+import {Modal} from '../../components';
+import {ColumnBox} from '../../components/UI';
+import {List} from '../../components/UI';
+import {Form} from '../../components/UI';
+import {TOOLTIPS} from '../../constants';
+import {connectServices} from 'superdesk-core/scripts/core/helpers/ReactRenderAsync';
+import {cloneDeep} from 'lodash';
+import eventsPlanning from '../../actions/eventsPlanning';
+
+class EventTemplateEdit extends React.Component {
+    constructor(props) {
+        super(props);
+
+        this.state = cloneDeep(this.props.template);
+
+        this.handleFieldChange = this.handleFieldChange.bind(this);
+    }
+    handleFieldChange(field, nextValue) {
+        this.setState({
```

Comment:
It did indeed allow saving empty values, but this is not the line at fault. It only manages the internal state, which is fine being empty. I'll disable a submit button in order to prevent sending the empty string out of the component.